src/Editor/SpielEditor.tsx

```diff
@@ -36,7 +36,7 @@ const SpielEditor = (props) => {
                     theme: props.editorTheme, 
                     readOnly: false
                 }}
-                editorDidMount={editor => { setEditor(editor); }}
```

Comment:
We may have to add this back if we want to highlight syntax errors in the future